.github/workflows/android.yml

```diff
@@ -0,0 +1,67 @@
+name: Android Build Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        abi: ['armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64']
+    steps:
+    - uses: nttld/setup-ndk@v1
```

Comment:
Question)
The latest version of setup-ndk is v1.4.2 and the version you uses is v1.0.0, which is released in 2021. Is there any reason to use v1?

Reply:
https://github.com/nnstreamer/nnstreamer/blob/b251e9bc6ba87b66056d4bf2adc852038cb7e38e/.github/workflows/android.yml#L15

I used that version to use the same version as nnstreamer.